test/end-to-end/pages/dashboard/chooseApplication.js

```diff
@@ -9,6 +9,7 @@ module.exports = async function(language ='en') {
     const I = this;
     const dashboardContent = language === 'en' ? dashboardEn : dashboardCy;
     await I.checkPageUrl('app/steps/ui/dashboard');
+    await I.waitForElement('#main-content', testConfig.TestWaitForTextToAppear);
     const welshLinkText = await I.grabTextFrom('//a[@class =\'govuk-link language\']');
     console.log('Dash Board Link Name::-->' + welshLinkText);
 
```

Comment:
Has this logging been left in by accident?

Reply:
No, this was a deliberate change, it forces the step to wait for a page to load before looking for the language link